<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupshareDialog</class>
 <widget class="QDialog" name="groupshareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>16</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);
font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>批量分享</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>73</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(58, 140, 255);</string>
   </property>
   <property name="text">
    <string>链接及二维码已成功生成！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>450</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonclos">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>400</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>179</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>复制链接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>237</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>294</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>二维码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sav">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>286</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>71</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/mainwindow/pictures/system_success.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonclos2">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>12</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/mainwindow/pictures/exit_button.png);
background-color: rgb(238, 240, 245);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>18</y>
     <width>23</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);
border-image: url(:/new/mainwindow/pictures/system_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>131</y>
     <width>328</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>143</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>链接：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>18</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);</string>
   </property>
   <property name="text">
    <string>批量分享</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButtonclos</zorder>
  <zorder>pushButton_copy</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_sav</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButtonclos2</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
